tendrl/ceph_integration/objects/definition/ceph.py

```diff
@@ -72,6 +89,177 @@
+              - ECProfile.m
+            optional:
+              - ECProfile.plugin
+              - ECProfile.directory
+          name: "Create ec profile"
+          run: tendrl.ceph_integration.objects.ECProfile.atoms.create.Create
+          type: Create
+          uuid: 7a2df258-9b24-4fd3-a66f-ee346e2e3730
+        Delete:
+          enabled: true
+          help: "Delete ec profile"
+          inputs:
+            mandatory:
+              - ECProfile.name
+          name: "Delete ec profile"
+          run: tendrl.ceph_integration.objects.ECProfile.atoms.delete.Delete
+          type: Delete
+          uuid: 7a2df258-9b24-4fd3-a66f-ee346e2e3740
+      flows:
+        DeleteECProfile:
+          atoms:
+            - tendrl.ceph_integration.objects.ECProfile.atoms.delete.Delete
+          help: "Delete ec profile"
+          enabled: true
+          inputs:
+            mandatory:
+              - ECProfile.name
+          run: tendrl.ceph_integration.objects.ECProfile.flows.delete_ec_profile.DeleteECProfile
+          type: Delete
+          uuid: 4ac41d8f-a0cf-420a-b2fe-18761e07f3b9
+      attrs:
+        name:
+          help: Name of the ec profile
+          type: String
+        k:
+          help: k value for ec profile
+          type: int
+        m:
+          help: m value for ec profile
+          type: int
+        plugin:
+          help: ec profile plugin
+          type: String
+        directory:
+          help: directory for ec profile
+          type: String
+        ruleset_failure_domain:
+          help: rule set failure domain for ec profile
+          type: String
+
+      enabled: true
+      list: clusters/$TendrlContext.integration_id/ec_profiles
+      value: clusters/$TendrlContext.integration_id/ec_profiles/$ECProfile.name
+      help: EC profile
+
+    Rbd:
+      atoms:
+        Create:
+          enabled: true
+          help: "Create rbd"
+          inputs:
+            mandatory:
+              - Rbd.pool_id
+              - Rbd.name
+              - Rbd.size
+          name: "Create Rbd"
+          run: tendrl.ceph_integration.objects.Rbd.atoms.create.Create
+          type: Create
+          uuid: 7a2df258-9b24-4fd3-a66f-ee346e2e3720
+        Delete:
+          enabled: true
+          help: "Delete rbd"
+          inputs:
+            mandatory:
+              - Rbd.pool_id
+              - Rbd.name
+          name: "Delete Rbd"
+          run: tendrl.ceph_integration.objects.Rbd.atoms.delete.Delete
+          type: Delete
+          uuid: 7a2df258-9b24-4fd3-a66f-ee346e2e3721
+        Resize:
+          enabled: true
+          help: "Resize Rbd"
+          inputs:
+            mandatory:
+              - Rbd.pool_id
+              - Rbd.name
+              - Rbd.size
+          name: "Resize Rbd"
+          run: tendrl.ceph_integration.objects.Rbd.atoms.resize.Resize
+          type: Update
+          uuid: 7a2df258-9b24-4fd3-a66f-ee346e2e3722
+      flows:
+        CreateRbd:
```

Comment:
It was discussed with @brainfunked and @anivargi and they only suggested to have this under `Rbd` object itself. I had a feeling as we always create rbds on top of pools, rather it should be under `Pool` object.